plugins/shared/annotate/index.js

```diff
@@ -168,6 +168,14 @@ module.exports = (namespace) => {
             });
         },
 
+        hide() {
+            $(".tota11y-label").hide();
```

Comment:
I initially tried using `ANNOTATION_CLASS`, but it didn't work. The annotations don't have that class.